### PR TITLE
manpage: add some minor documentation additions

### PR DESCRIPTION
**Pull request by thebombzen**

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -379,11 +380,11 @@ Program Behavior
     options that contain the word ``scale``. The special string ``*`` lists
     all top-level options.
 
-``-v``
-    Increment verbosity level, one level for each ``-v`` found on the command
+``--v``
```
Comment (kevmitch) [inline DOCS/man/options.rst]: Even if this works, the usual convention is that single letter arguments only have one dash.

Comment (thebombzen) [reply DOCS/man/options.rst]: Alright, can undo.

Comment (kevmitch) [reply DOCS/man/options.rst]: fine to merge with this change

Comment (kevmitch) [reply DOCS/man/options.rst]: undoing that is . . .

**Comment by kevmitch** [comment]
merged

**Pull request closed without merging**